packages/circom/tests/circuits/simple_regex_decomposed.circom

```diff
@@ -0,0 +1,242 @@
+pragma circom 2.1.5;
+
+include "@zk-email/zk-regex-circom/circuits/regex_helpers.circom";
+
```

Comment:
This doesn't seem to have the "regex in plaintext" content your added at the top, which was really useful...

Reply:
Also if we could add a "reveal group" comment to the top, it would help people exactly reproduce it.

Reply:
ok added these